packages/app-project/src/components/ProjectHeader/components/ProjectTitle/ProjectTitle.js

```diff
@@ -1,51 +1,51 @@
 import { Anchor, Heading } from 'grommet'
-import { string } from 'prop-types'
+import Link from 'next/link'
+import { withRouter } from 'next/router'
 import React from 'react'
 import styled from 'styled-components'
 
+const StyledHeading = styled(Heading)`
+  text-shadow: 0 2px 2px rgba(0, 0, 0, 0.22);
+`
+
 const StyledAnchor = styled(Anchor)`
   border-bottom: 3px solid transparent;
-  &:focus,
   &:hover {
```

Comment:
Why was the `:focus` styling removed?

Reply:
`Anchor` doesn't have any _additional_ `focus` styling ([src](https://github.com/grommet/grommet/blob/master/src/js/components/Anchor/StyledAnchor.js#L27)) - it inherits from `focusStyle` ([src](https://github.com/grommet/grommet/blob/master/src/js/utils/styles.js#L148)). I was minded to leave it without so we're at least handling focus consistently.

Reply:
Good point about consistent focus styling. I don’t feel strongly enough about the Grommet style to change it.